Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'sprockets-rails'
 group :assets do
   gem 'coffee-rails', '~> 5.0'
   gem 'dartsass-rails', '~> 0.5.1'
+  gem 'sass-embedded'
```

Comment:
how about this one, why do we need it?

Reply:
This was my failed attempt to fix the issue in midstream - I somehow thought that if it would be available in the cache, downloading from GitHub would not be required... I was wrong.